### DISCUSSION TITLE
Is investback.vercel.app a valid and compliant site using vercel?

### DISCUSSION THREAD

**Discussion by CryptoHobbyist** [vercel/community#1191] [Help] [answered]
### Summary

category:Help Is this a vailid site using the vercel.com product? The vercel.app domain just forwards to vercel.com. https://investback.vercel.app/ 

The site is an unfinished template and is asking for emails and to buy their crypto tokens.

**Comment by swarnava** [comment, marked as answer]
Please report abuse@vercel.com with detailed evidence if you think this is a Abuse website.

**Reply by CryptoHobbyist** [reply]
Thank you and I did. I also just found out that the website email they listed for their token is fake. support@investback.vercel.app
